python/cuml/test/test_api.py

```diff
@@ -119,14 +170,51 @@ def test_get_tags(model):
     return True
 
 
-def test_dynamic_tags():
-    estimator = dummy_estimator()
+def test_dynamic_tags_and_composition():
+    static_tags = dummy_class_with_tags._get_tags()
+    dynamic_tags = dummy_class_with_tags()._get_tags()
+    print(dummy_class_with_tags.__mro__)
+
+    # In python, the MRO is so that the uppermost inheritted class
+    # being closest to the final class, so in our dummy_class_with_tags
+    # the F Major input mixin overwrites the C mixin
+    assert static_tags['preferred_input_order'] == 'F'
+    assert dynamic_tags['preferred_input_order'] == 'F'
```

Comment:
In this case, wouldnt it be better to error out since this clearly would be an error? Or are you testing this for overriding purposes?

Reply:
Yeah, this test to be sure that the closest thing to the class in the MRO overwrites the farthest ones, added a quick comment to separate both tests and a quick note on the estimator guide about the MRO resolution and how that can affect tags.